conda_recipe_manager/parser/recipe_parser_convert.py

```diff
@@ -238,35 +282,16 @@ def _upgrade_build_section(self, base_package_paths: list[str]) -> None:
             # Canonically sort this section
             self._sort_subtree_keys(build_path, V1_BUILD_SECTION_KEY_SORT_ORDER)
 
-    def _upgrade_about_section(self) -> None:
+    def _upgrade_about_section(self, base_package_paths: list[str]) -> None:
         """
         Upgrades/converts the `about` section of a recipe file.
+        :param base_package_paths: Set of base paths to process that could contain this section.
         """
-        # Warn if "required" fields are missing
-        about_required: Final[list[str]] = [
-            "summary",
-            "description",
-            "license",
-            "license_file",
-            "license_url",
-        ]
-        for field in about_required:
-            path = f"/about/{field}"
-            if not self._new_recipe.contains_value(path):
-                self._msg_tbl.add_message(MessageCategory.WARNING, f"Required field missing: {path}")
-
-        # Transform renamed fields
-        about_rename: Final[list[tuple[str, str]]] = [
+        about_rename_mapping: Final[list[tuple[str, str]]] = [
```

Comment:
How helpful has Final been for you?

Reply:
Not as much as I would like. It sucks that you can't use it in a `for` loop. The static analyzer enforces it but it isn't as useful or really does much compared to C++'s const correctness or rust's immutability concepts.